README.md

```diff
@@ -26,6 +26,14 @@ crafter-cli create-site -e local -s my-site -n My Site \
 
 `crafter-cli sync-from -e local -s editorial -n origin`
 
+To create a group:
+
+`crafter-cli create-group -e local -gn test-group -gd "Description of the group"`
+
+Also, you can create a group by providing a csv file with the group name and description:
```

Comment:
You can bulk import groups by providing a CSV file with the format: `group name, group description`: